assets/js/components/notifications/CTA.js

```diff
@@ -76,6 +76,7 @@ CTA.propTypes = {
 	'aria-label': PropTypes.string,
 	error: PropTypes.bool,
 	onClick: PropTypes.func,
+	ctaLinkExternal: PropTypes.bool,
```

Comment:
Thanks for adding this :+1:. Could you please move it to go after the `ctaLink` prop?

Reply:
Sure.